a11y/overlays/index.md

```diff
@@ -0,0 +1,115 @@
+Утверждение о том, что оверлеи хороши для преодоления барьеров доступности, игнорирует тот факт, что пользователям, которым нужны эти функции на конкретном сайте, они понадобятся для запуска компьютера, открытия браузера и перехода на сайт. Хотя использование оверлея может улучшить соответствие нескольким положениям основных стандартов доступности, полного соответствия с помощью оверлея достичь невозможно.
+
+Добавление оверлея на сайт может противоречить предпочтениям конечных пользователей в отношении конфиденциальности и создавать риск несоблюдения применимых правил конфиденциальности. Некоторые оверлеи собирают данные и отправляют их поставщику.
+
+Ни один продукт-накладка на рынке не может сделать веб-сайт полностью соответствующим любому существующему стандарту доступности и, следовательно, не может устранить юридические риски в тех странах и регионах, где обеспечение доступности это обязанность бизнеса.
+В маркетинге поставщиков оверлеев часто звучит фраза о том, что `"достаточно одной строки кода"`. Они утверждают, что, добавив одну строку (JavaScript-кода), содержащую код оверлея, сайт станет доступным. Далее, как показано ниже, утверждается, что больше ничего не нужно менять на сайте клиента. Это утверждение было бы верным, если бы продукты делали то, что, по словам продавцов, они делают, но это не так. На самом деле было доказано, что такие продукты не могут устранять проблемы с доступностью в следующих типах контента:
+
+- Изображения и другой нетекстовый контент
+- Формы
+- Структура заголовка документа
+- Доступность клавиатуры и программное управление фокусом
+
+Настройки размера шрифта часто не увеличиваются до одного и того же уровня. Например, в `Allyable`, `AudioEye` и `UserWay` он увеличивается только до 200%, а в `EqualWeb` и `MaxAccess` — только до 150%. Встроенные функции для Windows и macOS позволяют увеличить его намного сильнее. Ни один из режимов высокой контрастности оверлеев не предлагает такого разнообразия, детализации или возможностей настройки, как в `Windows` или `macOS`.
+
+В последние годы в странах ЕС и США новые законы и судебные иски привели к широкому признанию того, что большинство частных компаний должны соблюдать эти требования. Для тех, кто их не соблюдает и получает жалобы, штрафы могут быть болезненными. Поэтому предоставление решения этой проблемы — очень выгодная бизнес-модель, особенно если вы можете сделать это быстро, дёшево и с минимальными изменениями в существующем пользовательском интерфейсе. Это ложное обещание, которое дают поставщики оверлеев.
```

Comment:
Тут можно написать, что оверлеи в странах ЕС официально сайты с оверлеями не соответствуют требованиям стандарта EN 301549, Директивы о веб-доступности (Web Accessibility Directive) и скоро вступающему в силу Европейскому закону о доступности (European Accessibility Act). Последний обязует все компании и организации соответствовать WCAG 2.1 на уровне AA.

> In December 2023, the European Commission made an [official statement on accessibility overlays and EAA compliance](https://www.edf-feph.org/publications/joint-statement-on-accessibility-overlays/):

> Claims that a website can be made fully compliant without manual intervention are not realistic, since no automated tool can cover all the WCAG 2.1 level A and AA criteria. It is even less realistic to expect to detect automatically the additional EN 301549 criteria.

Reply:
А ещё было бы клёво написать абзац про то, что пользователи доступных интерфейсов не в восторге от оверлеев.

>[AccessiBe got banned](https://nfb.org/about-us/press-room/national-convention-sponsorship-statement-regarding-accessibe) in 2021 by the National Federation of the Blind (NFB). A leading advocacy organization for the blind and visually impaired has had enough of their practices and the negative impact on the community.